website/src/models/user.js

```diff
@@ -328,6 +348,7 @@ export let schema = new Schema({
     orderAscending: {type: String, default: 'ascending'},
     quest: {
       key: String,
+      // TODO why are we storing quest progress here too and not only on party object?
```

Comment:
Yes. This is the user's progress throughout the day, not the total progress. It gets applied and cleaned up at cron.